config/localdevice/metac.yaml

```diff
@@ -1,60 +0,0 @@
-apiVersion: metac.openebs.io/v1alpha1
```

Comment:
I wanted us to use this file only. With 2 controllers. One controller for v1alpha1 & other for the upcoming v1.